.github/workflows/ci.yml

```diff
@@ -1,21 +1,32 @@
+# On branch push, download the windows testing bundle build and run it.
+
 name: cardano-wallet Windows Tests
 
 on:
   push:
     branches:
       - master
+      - bors/staging
+      - bors/trying
+  pull_request:
+    types: [opened, synchronize, reopened]
```

Comment:
| type | semantic |
| --- | --- | 
| opened | pull request is created |
| synchronize | commit(s) pushed to the pull request |
| reopened | closed pull request is reopened | 

(source: https://frontside.com/blog/2020-05-26-github-actions-pull_request/#how-do-workflows-trigger-on-pull_request)

Minor remark: maybe 'reopened' isn't needed here :thinking: ? If it's closed, it also means that tests were run previously already.

Reply:
From the same source :point_up: :
_When a Github Actions workflow is configured to run on pull requests, by default it will be triggered on three different types of activities: opened, synchronize, and reopened._

```
name: Pull Request Workflow
on: pull_request
```
I think default is fine, and no need to specify types. :man_shrugging:

Reply:
Yes you're right, I was just getting very confused by the docs.